### PR TITLE
Multiple follow up actions

### PR DESCRIPTION
**Pull request by ocofaigh**
### Description

- Updated to latest COS module (which has fix for existing COS instance bug)
- Added DA outputs
- Added some extra tests for existing resource use cases

Issue: https://github.com/terraform-ibm-modules/terraform-ibm-scc-da/pull/24

### Release required?

- [ ] No release
- [x] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: tests/resources/existing-resources/main.tf [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  retention_enabled      = false # disable retention for test environments - enable for stage/prod
+  kms_encryption_enabled = false
+}
+
+##############################################################################
+# Cloud Monitoring
+##############################################################################
+
+module "cloud_monitoring" {
+  source                  = "terraform-ibm-modules/observability-instances/ibm//modules/cloud_monitoring"
```
Comment (tyao117) [inline tests/resources/existing-resources/main.tf:34]: extra slash
```suggestion
  source                  = "terraform-ibm-modules/observability-instances/ibm/modules/cloud_monitoring"
```

File: tests/resources/existing-resources/main.tf [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+# EN
+##############################################################################
+
+module "event_notifications" {
+  source            = "terraform-ibm-modules/event-notifications/ibm"
+  version           = "1.2.2"
+  resource_group_id = module.resource_group.resource_group_id
+  name              = "${var.prefix}-en"
+  tags              = var.resource_tags
+  plan              = "lite"
```
Comment (tyao117) [inline tests/resources/existing-resources/main.tf:53]: should be a `var` that can be optional.

**Review by tyao117 — APPROVED** [review]
one minor thing, but since everything seems to be good in terms of testing.

File: tests/resources/existing-resources/main.tf (hunk 1 above)
Comment (ocofaigh) [reply tests/resources/existing-resources/main.tf:34]: @tyao117 The extra slash is the correct syntax required to call a submodule. See https://github.com/terraform-ibm-modules/terraform-ibm-observability-da/pull/2

**Pull request merged**